packages/block-editor/src/components/block-settings-menu/style.scss

```diff
@@ -1,59 +1,7 @@
-.block-editor-block-settings-menu__toggle .dashicon {
+.block-editor-block-settings-menu .components-dropdown-menu__toggle .dashicon {
 	transform: rotate(90deg);
 }
 
-// Popout menu
-.block-editor-block-settings-menu__popover {
-	&::before,
-	&::after {
-		margin-left: 2px;
-	}
-
-	.block-editor-block-settings-menu__content {
-		padding: ($grid-size - $border-width) 0;
-	}
-
-	.block-editor-block-settings-menu__separator {
-		margin-top: $grid-size;
-		margin-bottom: $grid-size;
-		margin-left: 0;
-		margin-right: 0;
-		border-top: $border-width solid $light-gray-500;
-
-		// Check if the separator is the last child in the node and if so, hide itself
-		&:last-child {
-			display: none;
-		}
-	}
-
-	.block-editor-block-settings-menu__title {
-		display: block;
-		padding: 6px;
-		color: $dark-gray-300;
-	}
-
-	// Menu items
-	.block-editor-block-settings-menu__control {
-		width: 100%;
-		justify-content: flex-start;
-		background: none;
-		outline: none;
-		border-radius: 0;
-		color: $dark-gray-500;
-		text-align: left;
-		cursor: pointer;
-		@include menu-style__neutral;
-
-		&:hover:not(:disabled):not([aria-disabled="true"]) {
-			@include menu-style__hover;
-		}
-
-		&:focus:not(:disabled):not([aria-disabled="true"]) {
-			@include menu-style__focus;
-		}
-
-		.dashicon {
-			margin-right: 5px;
-		}
-	}
+.block-editor-block-settings-menu .components-icon-button.has-text svg {
```

Comment:
What is the purpose of this rule? I can't see it applied.

Reply:
> What is the purpose of this rule? I can't see it applied.

Hmm, is it something I had added? Honestly, I cannot recall if or why it would be needed.